tests/layer_tests/tensorflow_tests/test_tf_Atan2.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
+
+class TestAtan2(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        assert 'y' in inputs_info
+        assert 'x' in inputs_info
+        y_shape = inputs_info['y']
+        x_shape = inputs_info['x']
+        inputs_data = {}
+        inputs_data['y'] = np.random.rand(*y_shape).astype(self.input_type)
+        inputs_data['x'] = np.random.rand(*x_shape).astype(self.input_type)
```

Comment:
@rghvsh, here you tests only one case when `x>=0 && y>=0`. Please add test other cases

Reply:
@rkazants I fixed this by adding negative of random.rand with same shape so there will be two cases 
case1: positive is greater or equal , then we get positive
case 2: negative is greater we get negative

Is this method fine?

Reply:
Yes, thanks!